### PR TITLE
Krishangpantey

### PR DESCRIPTION
**Pull request by krishangpatney**
base.html needs to be edited properly, pushed on a wrong change :)

### CODE REVIEW

File: wad_project/templates/socshare/base.html [hunk 1]
```diff
@@ -40,13 +40,19 @@
...
+                </ul>
+
             </nav>
         
             <div class="content {% if fullscreen %}fullscreen{% endif %}">
                 {% if not fullscreen %}
                 <div class="header">
-                    <button onclick="togglesideBar()">AAA</button>
-                    <b>{{ title }}</b>
+                    <button onclick="togglesideBar()" class="btn btn-light"> .. / {{ title }}</button>
```
Comment (ProstoSanja) [inline wad_project/templates/socshare/base.html]: I disagree

Comment (ProstoSanja) [reply wad_project/templates/socshare/base.html]: Why?

File: wad_project/templates/socshare/events.html [hunk 2]
```diff
@@ -1,14 +1,21 @@
 {% extends 'socshare/base.html' %}
 {% load static %}
 
+{% block title %}
+Events
+{% endblock %}
```
Comment (ProstoSanja) [inline wad_project/templates/socshare/events.html]: This block isn't necessary

File: wad_project/templates/socshare/events.html [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+
 {% block container %}
+<h1> Current Events </h1>
```
Comment (ProstoSanja) [inline wad_project/templates/socshare/events.html]: This is redundant af

File: wad_project/templates/socshare/profile.html [hunk 4]
```diff
@@ -7,6 +7,10 @@
     <img src="{{ MEDIA_URL }}{{ logo }}" class="logo" alt="">
     <h3 class="logo-name">{{ name }}</h3>
 </div>
+<br>
+<center><h2> Events </h2></center>
```
Comment (ProstoSanja) [inline wad_project/templates/socshare/profile.html]: Why?

File: wad_project/templates/socshare/society.html [hunk 5]
```diff
@@ -2,3 +2,53 @@
 {% load static %}
 {% load template_tags %}
 
+{% block title %}
+Societies
+{% endblock %}
```
Comment (ProstoSanja) [inline wad_project/templates/socshare/society.html]: Unnecessary block

File: wad_project/templates/socshare/base.html (hunk 1 above)
Comment (ProstoSanja) [reply wad_project/templates/socshare/base.html]: we decidede to not have ../

also #11 changes that

Comment (krishangpatney) [reply wad_project/templates/socshare/base.html]: Mainly cause its better than "aaa" and our wireframe had something very similar there but in the end, it's only a placeholder and needs to be changed regardless.

File: wad_project/templates/socshare/events.html (hunk 3 above)
Comment (krishangpatney) [reply wad_project/templates/socshare/events.html]: Can be removed, was added in the hopes that I could get past and current events separated.

File: wad_project/templates/socshare/profile.html (hunk 4 above)
Comment (krishangpatney) [reply wad_project/templates/socshare/profile.html]: There needs to be more information added into the profile. So thats a separator :)

File: wad_project/templates/socshare/society.html (hunk 5 above)
Comment (krishangpatney) [reply wad_project/templates/socshare/society.html]: Those blocks correspond to the ../societies button

File: wad_project/templates/socshare/events.html (hunk 3 above)
Comment (ProstoSanja) [reply wad_project/templates/socshare/events.html]: 👍

File: wad_project/templates/socshare/base.html (hunk 1 above)
Comment (ProstoSanja) [reply wad_project/templates/socshare/base.html]: fully included all of this in #11 now

**Comment by ProstoSanja** [comment]
do not merge before #11 and #17

File: wad_project/socshare/urls.py [hunk 6]
```diff
@@ -17,5 +17,7 @@
     path('test/', views.test, name='test'),
     path('logout/', views.logout_page, name='logout'),
     path('dashboard/add_event', views.add_event, name='add_event'),
-    path('dashboard/remove_event/<slug:event_slug>', views.remove_event, name='remove_event')
+    path('dashboard/<slug:event_slug>', views.edit_event, name = 'edit_event'),
+    path('dashboard/remove_event/<slug:event_slug>', views.remove_event, name='remove_event'),
+    path('soceity/', views.soceity, name='soceity')
```
Comment (fork-bombed) [inline wad_project/socshare/urls.py]: Typo, should be "society"

File: wad_project/socshare/views.py [hunk 7]
```diff
@@ -8,15 +8,35 @@
...
 from django.template.defaultfilters import slugify
 from django.db.models.functions.datetime import datetime
+from django.utils import timezone
 
 def events(request):
     search = request.GET.get('search')
-    events = Event.objects.filter(name__icontains=search) if search else Event.objects.order_by('date')
-    context = {"title":"Events","events":[x for x in events]}
+    eventsSearched = Event.objects.filter(name__icontains=search) if search else Event.objects.order_by('date')
+    context = {"title":"Events","events":[x for x in eventsSearched]}
```
Comment (fork-bombed) [inline wad_project/socshare/views.py]: Unnecessary name change and doesn't follow PEP8 naming conventions

File: wad_project/socshare/views.py [hunk 8]
```diff
@@ -8,15 +8,35 @@
...
+    context = {"title":"Events","events": event}
+    return render(request,'socshare/event.html',context=context)
+
+def edit_event(request, event_slug):
+    event = Event.objects.filter(slug = event_slug) 
+    context = {"title":"Events","events": event}
+    return render(request, 'socshare/edit_event.html', context)
+
+
+def soceity(request):
```
Comment (fork-bombed) [inline wad_project/socshare/views.py]: Typo, should be "society"

File: wad_project/templates/socshare/base.html [hunk 9]
```diff
@@ -30,7 +30,7 @@
                 <ul class="list-unstyled nav-links">
                     <li><a href="{% url 'socshare:events' %}">Events</a></li>
                     <li><a href="{% url 'socshare:calendar' %}">Calendar</a></li>
-                    <li><a href="{% url 'socshare:profile' 'test' %}">Societies</a></li>
+                    <li><a href="{% url 'socshare:soceity'%}">Societies</a></li>
```
Comment (fork-bombed) [inline wad_project/templates/socshare/base.html]: Typo, should be society

File: wad_project/socshare/views.py (hunk 7 above)
Comment (krishangpatney) [reply wad_project/socshare/views.py]: Yep, That shouldve gone back to events, my bad

**Comment by ProstoSanja** [comment]
force rebase------

**Pull request merged**